elasticdl/python/common/args.py

```diff
@@ -455,6 +455,14 @@ def add_common_args_between_master_and_worker(parser):
         help="The data reader parameters in a string separated by semi-colon "
         'used to instantiate the data reader, e.g. "param1=1; param2=2"',
     )
+    parser.add_argument(
+        "--distribution_strategy",
+        type=str,
+        default="",
```

Comment:
a better default value?

Reply:
"ParameterServerStrategy"? The only supported distribution strategy for the moment.

Reply:
yes, need a working default value.

Reply:
"master center storage" may not be accurate. The embedding parameters will locate in Parameter Servers. The difference is that the parameters of keras.Embedding table will all locate a single pod and the parameters of elasticdl.Embedding table will distribute all PS pods.

Reply:
If we want to have "master center storage" after the new PS implementation, we can create a [ParameterService](https://github.com/sql-machine-learning/elasticdl/blob/9bb018f3756ac3f707b44524d4fadc64e15080e2/elasticdl/python/ps/parameter_server.py#L17) instance in master.
Argument "num_ps_pods" can be used for ParameterServerStrategy only.